### PR TITLE
github: template: Add bugs

### PR DESCRIPTION
**Pull request by patrickelectric**
Github issue template to report a bug

Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

**Review by Williangalvani — APPROVED** [review]

**Comment by patrickelectric** [comment]
I'll wait for @ES-Alexander review before merging it

File: .github/ISSUE_TEMPLATE/bugs.yml
Comment (ES-Alexander) [inline .github/ISSUE_TEMPLATE/bugs.yml:9]:
```suggestion
        Thanks for taking the time to report an issue! Please fill out the following details so we can respond and act efficiently.
```

File: .github/ISSUE_TEMPLATE/bugs.yml
Comment (ES-Alexander) [inline .github/ISSUE_TEMPLATE/bugs.yml:32]:
```suggestion

- type: textarea
  attributes:
  label: Primary pain point(s)
  description: What is this bug actively stopping you from doing? This is helpful for us to suggest temporary workarounds while the actual issue is being fixed.
  placeholder: |
    Example: "No internet means I can't download new vehicle firmware"
  validations:
    required: false

```
I figure it's better to ask for this early as an option rather than having to ask for it in a response.

File: .github/ISSUE_TEMPLATE/bugs.yml
Comment (ES-Alexander) [inline .github/ISSUE_TEMPLATE/bugs.yml:37]:
```suggestion
      description: Please confirm the following (and check the boxes) before submitting.
```

File: .github/ISSUE_TEMPLATE/bugs.yml
Comment (ES-Alexander) [inline .github/ISSUE_TEMPLATE/bugs.yml:39]:
```suggestion
        - label: I have checked to make sure that a similar request has not already been filed or fixed.
```

**Review by ES-Alexander — CHANGES_REQUESTED** [review]
Mostly similar to the comments in #758, plus a suggestion to add a section for pain points. Review context is same as for #758.

**Review by ES-Alexander — APPROVED** [review]

**Pull request merged**
